core/templates/dev/head/player/PlayerServices.js

```diff
@@ -362,6 +406,18 @@ oppia.factory('oppiaPlayerService', [
     getExplorationId: function() {
       return _explorationId;
     },
+    getInformationCardData: function() {
+      informationCardData = {
```

Comment:
Should have a 'var' (or you can just remove the intermediate variable and return the object directly).